compiler/src/lnntoamm/Types.ts

```diff
@@ -260,16 +482,66 @@ export class FunctionType extends Type {
     this.retTy = retTy;
   }
 
-  static matrixSelect(fns: Fn[], args: Type[], scope: Scope): [Fn, Type][] {
-    const originalLength = fns.length;
+  /*
+  Original design comment (was written as a FIXME above `Expr.Call.inline`):
```

Comment:
Do we still need this comment?

Does it still depend on the order of `root.lnn`?

Reply:
Yes - I noted specifically that it's because of the ordering of `root.lnn` but realized that it's applicable to *all* code written in Alan, since the definition order of functions is important in determining selection and users may not realize that they need to change the ordering of their function declaration. I'll update the comment to reflect that